src/data/token.js

```diff
@@ -1,27 +1,33 @@
 import qs from 'qs'
 import { location, querystring, push } from 'svelte-spa-router'
 import { get as getStoreValue} from 'svelte/store'
-import { onClear, save, LIFESPAN, retrieve, clear, exists } from './storage'
+import { onClear, save, LIFESPAN, retrieve, clear, exists as isAlreadyInStorage } from './storage'
 import { loggedIn } from './analytics'
 
-let qsData = {}
+const get = key => retrieve(key) || qsData[key]
+export const seed = () => get('seed')
```

Comment:
I tend to prefer using some sort of verb in method names (since they should do something), and preserve noun-only names for non-methods, to make it clearer from even just the name whether we're looking at a method or just a value.

What would you think of renaming these methods to include a verb (e.g. `seed()` --> `getSeed()`, etc.)?

Reply:
sure, https://github.com/silinternational/wecarry-ui/pull/129/commits/c0e28e08dd7d27286c43f4d5983f07e28fa0688d